modules/python/src2/gen2.py

```diff
@@ -432,7 +432,11 @@ def __init__(self, atype, name, default_value, modifiers=(),
         self.tp = handle_ptr(atype)
         self.name = name
         self.defval = default_value
-        self._modifiers = tuple(modifiers)
+        # If the argument is Input/Output - remove Output only modifier
+        if '/IO' in modifiers:
+            self._modifiers = tuple(filter(lambda s: s != '/O', modifiers))
+        else:
+            self._modifiers = tuple(modifiers)
```

Comment:
Should this be done in hdr_parser.py instead?

Reply:
It is a good point, I hope there is no generators relying on presence of both modifiers...